client/src/css/style.css

```diff
@@ -59,55 +59,51 @@ body {
 }
 
 .carousel-content {
-  width: 80%;
+  width: 90%;
   padding-top:25px;
-
   margin:0 auto;
-  height:585px;
+  /* height:590px; */
+  display:flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  flex-wrap: nowrap;
+  align-items: flex-start;
+  align-content: center;
+
 }
 
 #carousel {
   display:inline-block;
   box-sizing: border-box;
-  width:95%;
+  width:90%;
```

Comment:
can set the buttons on both size "flex: 1 auto;" and then leave out the width property in here, and then in the parent div use "justify-content: stretch or  space-between"